cli/expense_commands.py

```diff
@@ -79,5 +80,18 @@ async def list_expenses(ctx, year=None, month=None, day=None):
         click.echo("No expenses found.")
         return
 
-    for expense in expenses:
-        click.echo(f"Category: {expense.category}, Amount: {expense.amount}, Date: {expense.date}")
+    if show_csv:
+        # Define the path where you want to save the CSV
+        csv_path = "expenses.csv"
```

Comment:
make a settings.py os.env var

Reply:
There's already a config.py that pulls from .env if necessary. I have added the csv_path there.

As for the headers, I have decided to leave it for now until I am done defining the core functionalities because model definitions will still change. e.g Expense model can still have a 'description' field etc.